.env.example

```diff
@@ -1,10 +1,25 @@
-TERRAFORM_S3_BUCKET="s3-bucket-name-here"
```

Comment:
Why remove these examples?

Reply:
It might mislead users like it's already set and they just skip configuration. And then quite complicated to find a mistake. If it's empty it's more obvious what is the problem. I faced that before.

Reply:
I think it's better to add comments with examples if needed

Reply:
Ok, I take your point. Maybe we can specify instructions (link to README if not set) like you did somewhere else in this project.

Reply:
Done